management_api_app/tests/test_api/test_routes/test_workspaces.py

```diff
@@ -102,9 +102,10 @@ async def test_workspaces_post_creates_workspace(create_workspace_item_mock, sav
 @patch("service_bus.service_bus.ServiceBus.send_resource_request_message")
 @patch("api.routes.workspaces.WorkspaceRepository.save_workspace")
 @patch("api.routes.workspaces.WorkspaceRepository.create_workspace_item")
-async def test_workspaces_post_calls_db_and_service_bus(create_workspace_item_mock, save_workspace_mock, send_resource_request_message_mock, app: FastAPI, client: AsyncClient):
+@patch("api.routes.workspaces.WorkspaceRepository._validate_workspace_parameters")
+async def test_workspaces_post_calls_db_and_service_bus(_validate_workspace_parameters_mock, create_workspace_item_mock, save_workspace_mock, send_resource_request_message_mock, app: FastAPI, client: AsyncClient):
     workspace_id = "000000d3-82da-4bfc-b6e9-9a7853ef753e"
-
+    _validate_workspace_parameters_mock.return_value = None
```

Comment:
nit: why _? It's not consistent with other mock names

Reply:
private vs public

Reply:
Fixed